### PR TITLE
add basic Obscuro Gateway user interface

### PR DESCRIPTION
**Pull request by zkokelj**
### Why this change is needed

To give users option to use Obscuro Gateway with 4 clicks from browser.

### What changes were made as part of this PR
New static page is served on `/og/`. It contains basic `html` and `js` that interacts with Obscuro Gateway endpoints and Metamask.

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/obscuronet/obscuro-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [ ] PR checks reviewed and performed

### CODE REVIEW

File: tools/walletextension/main/main.go [hunk 1]
```diff
@@ -75,6 +75,7 @@ func main() {
 
 	walletExtensionAddr := fmt.Sprintf("%s:%d", common.Localhost, config.WalletExtensionPortHTTP)
 	fmt.Printf("💡 Wallet extension started - visit http://%s/viewingkeys/ to generate an ephemeral viewing key.\n", walletExtensionAddr)
+	fmt.Printf("💡 Obscuro Gatway started - visit http://%s/og/ to use it.\n", walletExtensionAddr)
```
Comment (moraygrieve) [inline tools/walletextension/main/main.go:78]: Should read Gateway

File: tools/walletextension/api/staticOG/javascript.js [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+            params: [
+                {
+                    chainId: chainIdHex,
+                    chainName: 'Obscuro Testnet',
+                    nativeCurrency: {
+                        name: 'Obscuro',
+                        symbol: 'OBX',
+                        decimals: 18
+                    },
+                    rpcUrls: ['http://127.0.0.1:3000/?u='+userID],
```
Comment (zkokelj) [inline tools/walletextension/api/staticOG/javascript.js:43]: Get value of server where this app was deployed (local / dev-testnet / testnet / mainnet) and connect to the correct wallet extension.

File: tools/walletextension/api/staticOG/javascript.js [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+        if (accounts.length === 0) {
+            statusArea.innerText = "No MetaMask accounts found."
+            return
+        }
+        let authenticateAccountStatus = await authenticateAccountWithObscuroGateway(ethereum, accounts[0], userID)
+
+        statusArea.innerText = "\n Authentication status: " + authenticateAccountStatus
+
+         */
+        // show users an option to add another account and revoke userID
```
Comment (otherview) [inline tools/walletextension/api/staticOG/javascript.js:177]: Should this be removed ?

**Review by otherview — COMMENTED** [review]
LGTM just needs the server bind stuff - perhaps the golang manipulates the js file with the server address ?

**Review by otherview — APPROVED** [review]

**Pull request merged**